README.md

```diff
@@ -11,6 +11,20 @@ This API server enables audio transcription using the OpenAI Whisper models.
 
 # Build from source
 
+Before build make sure that **CGO_ENABLED** env is set to **1**
+
+```
+set CGO_ENABLED 1
+```
+
+or preferable set it parament. Then check it via
+
+```
+go env
+```
+
+Also you have to have installed gcc x64 i.e. by MYSYS
```

Comment:
I have GCC installed on my system, but I'm not sure if it's required. I will have to check this myself, thank you

Reply:
On my clean Windows 11 Pro installation I didn't get gcc preinstalled and when I tried to build after I setting the env variable I got error that gcc command couldn't be found